### DISCUSSION TITLE
Text formatting support in mermaid.js diagrams

### DISCUSSION THREAD

**Discussion by foobarbecue** [community/community#61266] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Github [supports the mermaid.js diagram system](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/), and it's great.

I can do things like this:
```mermaid
flowchart
  A-->B[sample text\nother text]
  B-->C[more text]
  B--back up to-->A
```

However, if I want to use bold or italic text (or other markdown features) inside this plot, there doesn't seem to be a way to do that.

Mermaid.js [supports markdown internally](https://mermaid.js.org/syntax/flowchart.html#markdown-formatting), but that syntax doesn't seem to work on github -- for example bold text ends up like this:

```mermaid
flowchart
  A-->B
  B-->C["`**should be bold**`"]
  B--back up to-->A
```

I'm wondering if this might be because the embedded version of mermaidjs on github is old.

Please support this syntax!

**Comment by sidharthv96** [comment]
Yes, the current version github uses is below, and markdown support was added in [v10.1.0](https://github.com/mermaid-js/mermaid/releases/tag/v10.1.0)
```mermaid
info
```
